Client/Frontend/Browser/Playlist/Controllers/PlaylistListViewController+TableViewDataSource.swift

```diff
@@ -76,7 +76,7 @@ extension PlaylistListViewController: UITableViewDataSource {
                 }
                 return nil
             }
-        } else {
+        } else if trackStatus != .loading {
```

Comment:
Dont want to log if it is failed?

Reply:
It will log only if it is `!loading` (not loading). So if it fails or gets cancelled or anything, it will log. But when it is `loading` it will not log because it spams the console with 100+ message that it is loading every single byte :(